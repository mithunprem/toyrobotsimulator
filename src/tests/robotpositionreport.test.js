import React from 'react';
import { create } from 'react-test-renderer';
import RobotPositionReport from '../Components/RobotPositionReport';

describe("RobotPositionReport", () => {
  const robotPosition = { x:1, y:1, direction:'NORTH' };
  test("should render without throwing an error", () => {
    const report = create(<RobotPositionReport robotPosition={robotPosition} />).toJSON()
    expect(report).toMatchSnapshot()
  });
});
