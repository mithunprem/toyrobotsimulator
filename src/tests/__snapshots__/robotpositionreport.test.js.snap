// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RobotPositionReport should render without throwing an error 1`] = `
<div
  className="report alert alert-info"
>
  <span>
    Robot's current position is: 
    1
    , 
    1
    , 
    NORTH
  </span>
</div>
`;
